can_tools/bootstrap_data/covid_categories.csv

```diff
@@ -1,5 +1,6 @@
 group,category
 case,cases
+case,monkeypox_cases
```

Comment:
This does stray us a bit away from the intention of this schema and I'm not a huge fan of doing it this way, but: 

Ideally, I think we'd want to spin up a new schema that's parallel to the `covid` schema for this monkeypox data (which would consist of creating a `monkeypox_observations` table and a `monkeypox_us` view in Postgres, or something similar), but I'm not sure that's worth the investment at the moment.

Reply:
Those schemas are created here: https://github.com/covid-projections/can-scrapers/blob/main/can_tools/models.py#L219

Reply:
Yeah, I think this is the right approach for now.  If we get serious about collecting monkeypox data (e.g. scraping state dashboards and/or related monkeypox metrics), then we should revisit.